internal/quickstart/messages.go

```diff
@@ -40,10 +43,14 @@ func toggleFlag(client flags.Client, accessToken, baseUri, flagKey string, enabl
 			flags.BuildToggleFlagPatch(defaultEnvKey, enabled),
 		)
 		if err != nil {
-			return errMsg{err: err}
+			var errorMsg errMsg
+			hasToggleConflict, errorMsg = handleError(err, "Error toggling flag")
+			if !hasToggleConflict {
+				return errorMsg
+			}
```

Comment:
I don't like that I'm checking the conflict variable here again. I don't have an alternative solution in mind for this though